net.nymtech.NymVPN.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=NymVPN
+Type=Application
+Version=1.0
+Comment=Decentralized, mixnet, and zero-knowledge VPN
+Exec=/app/bin/entry.sh %U
```

Comment:
this is NEVER an absolute path.

Reply:
also this doesn't match the `command` property from the manifest. It should.

Reply:
The purpose is to provide different env vars depending on if it is launched from terminal (`flatpak run net.nymtech.NymVPN`)  or via the desktop entry. As I would like to have LOG_FILE set when launching from the desktop, but not when launching from terminal.

Reply:
But let's keep it simple, I'll update to use the same wrapper script for both `command` and desktop.